feature-group-definitions/offscreencanvas.yml

```diff
@@ -0,0 +1,2 @@
+spec: https://html.spec.whatwg.org/multipage/#the-offscreencanvas-interface
```

Comment:
`offscreen-canvas` as two words maybe? I suspect that "offscreen canvas" (the concept) and its interface names aren't quite the same thing (supported by the quick and dirty test showing that there are more Stack Overflow questions with "offscreen canvas" in the title than "OffscreenCanvas").

Reply:
Looks like it is sometimes referred to as "offscreen canvas", so sure! Example: https://webrtc.ventures/2022/02/offscreen-canvas-api/